pom.xml

```diff
@@ -42,11 +42,11 @@
     </developers>
 
     <properties>
-        <gridsuite-dependencies.version>35</gridsuite-dependencies.version>
+        <gridsuite-dependencies.version>36</gridsuite-dependencies.version>
         <string-template.version>4.3.1</string-template.version>
         <liquibase-hibernate-package>org.gridsuite.filter.server</liquibase-hibernate-package>
         <sonar.coverage.exclusions>**/migrations/**/*</sonar.coverage.exclusions>
-        <filter.version>1.0.16</filter.version>
+        <filter.version>1.2.0</filter.version>
```

Comment:
Isn't it now included in gridsuite-dependencies ?

Reply:
Right
I have removed the version override